### ISSUE TITLE
Implement borrower happy path e2e tests

### ISSUE THREAD

**Issue by philipstanislaus** [centrifuge/e2e-tests#4]
Blocked by spec from @denniswell 

TC: 
* Only run manually for now, not on CI

**Comment by philipstanislaus** [comment]
Duplicate of https://github.com/CentrifugeInc/tinlake-roadmap/issues/22
